src/main/java/com/example/tournamentMaker/team/RegistrationCompleteException.java

```diff
@@ -0,0 +1,7 @@
+package com.example.tournamentMaker.team;
+
+class RegistrationCompleteException extends RuntimeException {
```

Comment:
ten wyjatek sie nazywa jakbys po skonczonej rejestracji rzucal wyjatkiem.
Nazwij to lepiej np po prostu TournamentRegistrationException albo cos